.github/workflows/standal-ce-nginx.yml

```diff
@@ -0,0 +1,71 @@
+name: 'standal-ce-nginx'
+
+on:
+  push:
+
+    # Only run on these branches
+    branches:
+      - main
+
+    paths:
+
+      # Implementation specific
+      - '.github/workflows/template.yml'
+      - 'docker/template/**'
```

Comment:
This should be `- 'docker/standal-ce-nginx/**'`. Sorry, will make this more clear in the template CI pipe :)

Reply:
Oh yeah, we need this change to trigger the workflow.

Reply:
No I think you're right, so only changes to this one trigger this one.

I completely glossed over every one of the "implementation specific" items 😬